packaging/docker/Dockerfile

```diff
@@ -96,13 +96,13 @@ RUN curl -Ls $FDB_WEBSITE/downloads/$FDB_VERSION/linux/fdb_$FDB_VERSION.tar.gz |
 
 # Install additional FoundationDB Client Libraries
 RUN for version in $FDB_LIBRARY_VERSIONS; do \
-    curl -Ls $FDB_WEBSITE/downloads/$version/linux/libfdb_c_$version.so -o /usr/lib/fdb/multiversion/libfdb_c_${version%.*}.so; \
+    curl -Ls $FDB_WEBSITE/downloads/$version/linux/libfdb_c_$version.so -o /usr/lib/fdb/multiversion/libfdb_c_${version}.so; \
```

Comment:
have we built images and looked at a side-by-side of how this turns out. The shell substring removal makes sense to me `${version%.*} to ${version}` but the pattern that `.*` is matching is not completely clear to me. Was the original intent to remove last version digit from the file name (i.e. `7.1.0` becomes `7.1`), and now we are keeping the complete version in the file name?

Reply:
Yes that's the exact intent of this. In the case of release candidates, we would end up with `7.1.0-rc1` -> `7.1`.